crates/oxc_codegen/src/gen.rs

```diff
@@ -3754,12 +3754,12 @@ impl Gen for TSEnumDeclaration<'_> {
         p.print_str("enum ");
         self.id.print(p, ctx);
         p.print_space_before_identifier();
-        p.print_curly_braces(self.span, self.members.is_empty(), |p| {
-            for (index, member) in self.members.iter().enumerate() {
+        p.print_curly_braces(self.span, self.body.members.is_empty(), |p| {
+            for (index, member) in self.body.members.iter().enumerate() {
```

Comment:
Could simplify this a bit by implementing `Gen` for `TSEnumBody`.

Also, is the span wrong?

```diff
- p.print_curly_braces(self.span, self.members.is_empty(), |p| {
+ p.print_curly_braces(self.body.span, self.members.is_empty(), |p| {
```

(that's a question, not a statement - I'm not very familiar with codegen)

Reply:
Thanks, I think both are true. Fixed in 31d09f1